doc/man3/EC_GROUP_copy.pod

```diff
@@ -177,6 +178,7 @@ specified curve respectively. If there is no curve name associated with a curve
 EC_GROUP_get0_order() returns an internal pointer to the group order.
 EC_GROUP_order_bits() returns the number of bits in the group order.
 EC_GROUP_get0_cofactor() returns an internal pointer to the group cofactor.
+EC_GROUP_get0_field() returns an internal pointer to the group field.
```

Comment:
I think this needs a better description. It doesn't tell us anything about how this value should be interpreted. For example compare with the text in ec_lcl.h about this value:

````
    /*
     * Field specification. For curves over GF(p), this is the modulus; for
     * curves over GF(2^m), this is the irreducible polynomial defining the
     * field.
     */
````

Reply:
I added the description and unit tests.